website/views/options/social/party/loading-new-party.jade

```diff
@@ -1,5 +1,5 @@
 .container-fluid
   .row.text-center: .col-xs-12
-    h2=env.t('loadingNewParty')
+    h2=env.t('partyLoading' + partyLoadingMessage)
```

Comment:
That didn't quite work, but it sent me in the right direction! Thanks, I didn't know client- vs server-side variables were a thing.